### PR TITLE
add Webzie.com / page builder

### PR DESCRIPTION
**Pull request by RivierGrullon**
## website
https://www.webzie.com/

### examples
https://www.dogs-cats.co.il/
https://www.chabad-judaica.com/
https://www.smilephone.co.il/
https://www.toys-express.co.il/
https://www.pashutrihut.co.il/
https://www.first-sport.co.il/
https://www.avinary-pro.com/
https://www.holylandlegacy.co.il/
https://www.deal-shop.co.il/
https://www.decoclick.co.il/
https://www.thebigmall.co.il/
https://www.anak-rihut.co.il/
https://www.pilotshop.co.il/
https://www.footballworld.co.il/
https://www.pumps4u.co.il/
https://www.emanueel.co.il/
https://www.at-work.co.il/
https://www.shakedsport.co.il/
https://www.zerimyafim.co.il/
https://www.yashir-la.co.il/

### CODE REVIEW

**Comment by RivierGrullon** [comment]
The check failed for me locally because there's a missing icon for a different technology (not this - it's for nopStation).

File: src/technologies/w.json [hunk 1]
```diff
@@ -850,6 +850,24 @@
...
+  "Webzie.com": {
+    "cats": [
+      1,
+      6,
+      92
+    ],
+    "description": "Webzie is a website builder optimized for performance.",
+
+    "meta": {
+      "generator": "Webzie\\.com Website Builder"
```
Comment (nurbek91) [inline src/technologies/w.json:862]: `"generator": "^Webzie\\.com\\sWebsite\\sBuilder$"`

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!!  
Fixed false-positive results   here #6420  that would interfere with this technology
Needs minor updates to bring it to perfection

**Comment by RivierGrullon** [comment]
@nurbek91 this technology is "[webize.com](http://webize.com/)" and there is another tech called "[webzie.ir](http://webzie.ir/)"

**Review by nurbek91 — APPROVED** [review]
Hey mate , we can leave the name as Webzie without.com

**Comment by RivierGrullon** [comment]
> Hey mate , we can leave the name as Webzie without.com

@nurbek91 done :)

**Comment by RivierGrullon** [comment]
@AliasIO this Technology is diferent than you commited #6420

**Pull request closed without merging**